autoload/lsp/ui/vim/completion.vim

```diff
@@ -107,7 +107,11 @@ function! s:on_complete_done_after() abort
     else
       let l:overflow_before = 0
       let l:overflow_after = 0
-      let l:text = get(l:completion_item, 'insertText', l:completed_item['word'])
+      let l:text = get(l:completion_item, 'insertText', '')
+      if empty(l:text)
+        " When insertText is `falsy` the label is used as the insert text
+        let l:text = get(l:completion_item, 'label', l:completed_item['word'])
+      endif
     endif
```

Comment:
Thank you.
In spec, the insertText can't be `null` but some servers seem to return it as null

Reply:
Hmm, I'm not familiar with the specification. But it says insertText can be 'falsy'. I thought `null` is included in the falsy values. Anyway `empty()` can handle both empty string and null so it should work well even if null is invalid for insertText.

Reply:
Ah. You are right.

Sorry. I have overlooked doc comments. (I had only seen type annotations).